posthog/batch_exports/service.py

```diff
@@ -70,6 +70,7 @@ class SnowflakeBatchExportInputs:
     table_name: str = "events"
     data_interval_end: str | None = None
     role: str | None = None
+    legacy_schema: bool = False
```

Comment:
yes, but it's a question about the future rather ... because we don't really want to default to something that has person_id as thats misleading with the way merges work ... potentially

Reply:
Yeah, that is a fair point. I've made legacy the default moving forward by setting this to True. It does mean we need to update exports when we merge.